packages/roosterjs-editor-dom/lib/edit/adjustInsertPosition.ts

```diff
@@ -161,11 +160,19 @@ function adjustInsertPositionForStructuredNode(
             );
         }
     } else if (tag == 'TABLE' && trNode) {
+        pasteTable(root, nodeToInsert, position, range);
```

Comment:
Instead of calling pasteTable() here, let's just add a new "adjustInsertPosition" function, say "adjustInsertPositionForTable".